services/account_type/cloudtrail_owner.yaml

```diff
@@ -1,73 +1,73 @@
-AWSTemplateFormatVersion: "2010-09-09"
+AWSTemplateFormatVersion: '2010-09-09'
 
 
-Description: "CloudZero :: CloudTrail owner template"
+Description: CloudZero CloudTrail Owner Template
 
 
 Parameters:
   IsCloudTrailOwnerAccount:
-    Description: "Flag that indicates if this account owns the cloudtrail"
-    Type: "String"
+    Description: Flag that indicates if this account owns the cloudtrail
+    Type: String
     AllowedValues:
-      - "true"
-      - "false"
+      - 'true'
+      - 'false'
   CloudTrailSNSTopicArn:
-    Description: "The CloudZero SNS Topic ARN responsible for receiving notifications from the cloudtrail"
-    Type: "String"
+    Description: 'The CloudZero SNS Topic ARN responsible for receiving notifications from the cloudtrail'
+    Type: String
   ReactorAccountId:
-    Description: "The CloudZero reactor AWS account ID"
+    Description: The CloudZero reactor AWS account ID
     Type: String
-    Default: "061190967865"
+    Default: '061190967865'
 
 
 Conditions:
-  CreateResources: !And [ !Not [ !Equals [ !Ref CloudTrailSNSTopicArn, "null" ] ], !Equals [ !Ref IsCloudTrailOwnerAccount, "true" ] ]
+  CreateResources: !And [ !Not [ !Equals [ !Ref CloudTrailSNSTopicArn, 'null' ] ], !Equals [ !Ref IsCloudTrailOwnerAccount, 'true' ] ]
 
 
 Resources:
 
   SqsQueue:
-    Type: "AWS::SQS::Queue"
+    Type: 'AWS::SQS::Queue'
     Condition: CreateResources
     Properties:
-      QueueName: !Join [ "-", [ "cz-reactor-events", !Ref ReactorAccountId ] ]
+      QueueName: !Sub "cz-reactor-events-${ReactorAccountId}"
       Tags:
         - Key: cloudzero-stack
           Value: !Ref AWS::StackName
         - Key: cloudzero-reactor-account-id
           Value: !Ref ReactorAccountId
 
   SnsTopic:
-    Type: "AWS::SNS::Topic"
+    Type: 'AWS::SNS::Topic'
     Condition: CreateResources
     Properties:
-      DisplayName: !Join [ "-", [ "cloudtrail-events", !Ref ReactorAccountId ] ]
-      TopicName: !Join [ "-", [ "cloudtrail-events", !Ref ReactorAccountId ] ]
+      DisplayName: !Sub "cloudtrail-events-${ReactorAccountId}"
```

Comment:
@jeremyhahn, please replace the `Sub` `"`s (double quotes) with `'`s (single quotes) in all places. Then you're good to go.